### ISSUE TITLE
Audit: Facilities Drupal content types for Phone number fields implementation

### ISSUE THREAD

**Issue by jilladams** [department-of-veterans-affairs/va.gov-cms#17814]
## Description or Additional Context 
Question we are trying to answer: Which phone numbers fields use Telephone number field alone, vs. Telephone field with a separate extension input. 

## Acceptance Criteria
- [x] Document where phone numbers are used and which type
- [x] Make notes of the help text associated with those fields
- [x] Link to notes on #16148

**Comment by davidmpickett** [comment]
@jilladams Adding to Sprint 1, but noting it was an injection

**Comment by laflannery** [comment]
Document with audit and summary:
[Facilities Phone Number field summary.docx](https://github.com/department-of-veterans-affairs/va.gov-cms/files/14962988/Facilities.Phone.Number.field.summary.docx)

**Comment by laflannery** [comment]
@jilladams After doing this and closing it, I'm unsure if there is a clear next step. Would I go through this at a UX sync? Does the ticket I links to in the last AC just pick it up as the next step? I just want to make sure the follow through is taken care of

**Comment by jilladams** [comment]
I think the right next step is some Product work to: 
* Review your notes
* Review the state of existing other tickets and reduce overall confusion
* Define a logical next step of work. 

I think you and I (and @davidmpickett if he's interested) could have a working meeting to look at a few things and get a lot of clarity.  #16148 is planned for next sprint. I'll look for time early in the week for us to chat.

**Comment by jilladams** [comment]
Doc is uploaded to Sharepoint here: https://dvagov.sharepoint.com/:w:/r/sites/SitewideContract/_layouts/15/Doc.aspx?sourcedoc=%7B0D398299-186D-47CD-B8AE-8532D70B552D%7D&file=Facilities.Phone.Number.field.summary.docx&action=default&mobileredirect=true

Super epic for phone number problems now exists: #17854. New tickets can go there.

**Comment by jilladams** [comment]
Laura's audit found 3 phone number fields in Drupal that need to be split with a separate extension field ( using the Phone number paragraph type field). We will have 7 tickets: a Drupal and a FE ticket for each of these (that will live in #17854). : 
1. Drupal & FE: Staff profile: number
2. Drupal & FE: Billing & Insurance number
3. Drupal & FE: VAMC Facility mental health number
    * This ticket for Drupal should have an AC to verify the migration push with Lighthouse before merging. 
4. The phone paragraph type validation needs to add validation to the extension field to prevent text entry (which breaks the FE component).

Optional things to consider separately / later: 
* Separately ticketing breaking up phone number fields that aren't currently editable to align them with the paragraph / separate extension field. Not urgent. Laura will sub a backlog ticket for this.
* 12 char max doesn't allow for int'l #s, but that's not a problem we need to solve today.  (Laura spot checked Guam, and they seem to be using 1-800 #s.)
* #15691 is an existing spike, where we can add some helpful notes.

**Comment by davidmpickett** [comment]
FYI Here's the Character Counter guidance issue:
https://github.com/department-of-veterans-affairs/va.gov-cms/issues/16659

**Comment by jilladams** [comment]
@laflannery when you talk to Marisa about the proposed guidance here, could you two take a look at #15691 and note there whether you feel like that ticket is still necessary? (Relevant bc right now it technically blocks #16906. If we decide the provided guidance is Good Enough For Now TM, then #16906 might become unblocked.)

**Comment by laflannery** [comment]
@jilladams I definitely can - is there a deadline or anything for this now that there is a VBA connection? I.E, we want to find out if this is unblocked by X date?

**Comment by jilladams** [comment]
@laflannery 16906 isn't considered launch blocking (according to launch plan https://github.com/department-of-veterans-affairs/va.gov-cms/issues/16268), so not on fire, just something we'd like to get to if/when possible for Pilot.
